Simulation/Classes/StateVector_Index.m

```diff
@@ -0,0 +1,24 @@
+%% State Vector Indecies
+% This is an enumeration of all indecies of a Vehicle State Vector
+
+% Note: enumerations are currently not implemented in Octave
+% (https://savannah.gnu.org/bugs/?44582) so constant properties are used instead.
+
+classdef StateVector_Index
+	properties (Constant)
+    P_X = 1 % Position, X
+    P_Y = 2 % Position, Y
+    P_Z = 3 % Position, Z
+    P   = [1,2,3] % Position, Vector [X,Y,Z]
+    V_X = 4 % Velocity, X
+    V_Y = 5 % Velocity, X
+    V_Z = 6 % Velocity, X
+    V   = [4,5,6] % Velocity, Vector [X,Y,Z]
+    O_P = 7 % Orientation, Pitch
+    O_R = 8 % Orientation, Roll
+    O_Y = 9 % Orientation, Yaw
+    O   = [7,8,9] % Orientation, Vector [Pitch,Roll,Yaw]
+    M   = 10 % Mass
+    B_Cap = 11 % Battery Capacity
+	end
+end

```

Comment:
As we discussed, we probably want to find a different solution for this. That's out of the scope of this PR so we should probably just open an issue.

Reply:
Unrelated but there seem to be some indentation issues here. It's a tab vs spaces issue and we should really just stick with one. There seems to be more of this mixing below so you should probably configure your code editor to do this automagically.

Reply:
I think I misunderstood one of the settings in the Atom editor settings about how it deals with tabs. I now explicitly tell it to only use spaces for tabs everywhere, hopefully that solves the issue (I don't think Octave has this issue, but we will see).

Reply:
I have removed any tabs and replaced them with spaces on this file